code/noRAM/app/src/main/java/com/example/noram/OrganizerEventAttendeeActivity.java

```diff
@@ -1,14 +1,43 @@
+/*
+* The OrganizerEventAttendeeActivity class displays the list of attendees for an event
+* and the number of times they have checked in.
+* Outstanding Issues:
+* - None
+ */
+
 package com.example.noram;
 
+import android.content.Intent;
 import android.os.Bundle;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.ListView;
 
 import androidx.appcompat.app.AppCompatActivity;
 
+import com.example.noram.controller.EventAttendeeArrayAdapter;
+import com.example.noram.model.AttendeeCheckInCounter;
+import com.example.noram.model.Event;
+import com.google.android.gms.tasks.Task;
+import com.google.firebase.firestore.DocumentSnapshot;
+
+import java.util.ArrayList;
+import java.util.Locale;
 /**
  * The OrganizerEventAttendeeActivity class displays the list of attendees for an event.
  * A {@link AppCompatActivity} subclass.
  */
 public class OrganizerEventAttendeeActivity extends AppCompatActivity {
```

Comment:
nit: OrganizerEventAttendeeListActivity is a bit more wordy, but is 10x easier to understand